docs/index.html

```diff
@@ -384,6 +420,30 @@ <h1>Electronics and electrical design checklist</h1>
 				<li>Is there filtering between analog and digital commons?</li>
 				<li>Are optocouplers filtered?</li>
 				<li>Impedance on inputs from outside of board?</li>
+				<li> Are ferrite beads on input/output power lines?</li>
+				<li> Are ferrite beads on sensitive signal lines? 
+					<div>
+					This includes signal lines that are not sensitive themselves, but connect to sensitive components.
+					E.g. U-Blox recommend ferrite beads on GPS module UART connections to avoid RF noise travelling "along" and into the module, see <a href="https://www.u-blox.com/sites/default/files/NEO-M8_HardwareIntegrationManual_%28UBX-13003557%29.pdf"> U-Blox M8 Hardware Integration Manual </a>
+					</div> 
+				</li>
+				<li> Do all ferrite beads have sufficient margin in DC current rating? 
+					<div>
+					When the DC current (DC bias current) approaches the maximum, the effective impedance of ferrite beads decreases significantly. See <a href="https://www.analog.com/media/en/technical-documentation/application-notes/an-1368.pdf"> Analog Devices AN-1368, Choosing and Using Ferrite Beads </a>
+					</div>
+				</li>
+				<li> Is there an estimate of what frequencies the ferrite beads are required to filter? </li>
+				<li> Do high speed single ended digital signals have series resistors? </li>
+				<li> Is there a footprint for a common mode filter on high speed differential signals going offboard?
+					<div>
+					For high speed differential buses like USB3.0, DisplayPort, and HDMI, a common mode filter will stop common mode noise generated in the transciever from travelling into the signal line where it may radiate from the cable, though will degrade the differential signal slightly.
```

Comment:
```suggestion
					For high-speed differential buses like USB 3.0, DisplayPort, and HDMI, a common-mode filter will stop common-mode noise generated in the transceiver from traveling into the signal line where it may radiate from the cable. Notice that the filter might though will degrade the differential signal slightly.
```

Reply:
Slight alteration, agree on the rest of the line
```suggestion
					For high-speed differential buses like USB 3.0, DisplayPort, and HDMI, a common-mode filter will stop common-mode noise generated in the transceiver from traveling into the signal line where it may radiate from the cable. Notice that the filter might degrade the differential signal slightly.
```